Libraries/src/Amazon.Lambda.AspNetCoreServer.Hosting/ServiceCollectionExtensions.cs

```diff
@@ -37,20 +39,42 @@ public static class ServiceCollectionExtensions
         /// <param name="eventSource"></param>
         /// <returns></returns>
         public static IServiceCollection AddAWSLambdaHosting(this IServiceCollection services, LambdaEventSource eventSource)
+        {
+            // Not running in Lambda so exit and let Kestrel be the web server
+            return services.AddAWSLambdaHosting(eventSource);
+        }
+
+        /// <summary>
+        /// Add the ability to run the ASP.NET Core Lambda function in AWS Lambda. If the project is not running in Lambda 
+        /// this method will do nothing allowing the normal Kestrel webserver to host the application.
+        /// </summary>
+        /// <param name="services"></param>
+        /// <param name="eventSource"></param>
+        /// <param name="configure"></param>
+        /// <returns></returns>
+        public static IServiceCollection AddAWSLambdaHosting(this IServiceCollection services, LambdaEventSource eventSource, Action<HostingOptions>? configure = null)
         {
             // Not running in Lambda so exit and let Kestrel be the web server
             if (string.IsNullOrEmpty(Environment.GetEnvironmentVariable("AWS_LAMBDA_FUNCTION_NAME")))
                 return services;
 
+            var hostingOptions = new HostingOptions();
+            
+            if (configure != null)
+                configure.Invoke(hostingOptions);
+
+            services.AddSingleton<ILambdaSerializer>(hostingOptions.Serializer ?? new DefaultLambdaJsonSerializer());
```

Comment:
Use `TryAddSingleton` just in case somebody gets the idea to add the `ILambdaSerializer` directly to the service collection.

Reply:
Done.